website/content/docs/runtime/interpolation.mdx

```diff
@@ -63,227 +63,37 @@ Below is a full listing of node attributes that are interpretable. These
-    </tr>
-    <tr>
-      <td>
-        <code>
-          ${'{'}attr.driver.&lt;property&gt;{'}'}
-        </code>
-      </td>
-      <td>
-        See the <a href="/docs/drivers">task drivers</a> for property
-        documentation
-      </td>
-    </tr>
-    <tr>
-      <td>
-        <code>{'${attr.unique.hostname}'}</code>
-      </td>
-      <td>Hostname of the client</td>
-    </tr>
-    <tr>
-      <td>
-        <code>{'${attr.unique.network.ip-address}'}</code>
-      </td>
-      <td>
-        The IP address fingerprinted by the client and from which task ports are
-        allocated
-      </td>
-    </tr>
-    <tr>
-      <td>
-        <code>{'${attr.kernel.arch}'}</code>
-      </td>
-      <td>
-        Kernel architecture of the client (e.g. <code>x86_64</code>, <code>aarch64</code>)
-      </td>
-    </tr>
-    <tr>
-      <td>
-        <code>{'${attr.kernel.name}'}</code>
-      </td>
-      <td>
-        Kernel of the client (e.g. <code>linux</code>, <code>darwin</code>)
-      </td>
-    </tr>
-    <tr>
-      <td>
-        <code>{'${attr.kernel.version}'}</code>
-      </td>
-      <td>
-        Version of the client kernel (e.g. <code>3.19.0-25-generic</code>,{' '}
-        <code>15.0.0</code>)
-      </td>
-    </tr>
-    <tr>
-      <td>
-        <code>{'${attr.platform.aws.ami-id}'}</code>
-      </td>
-      <td>AMI ID of the client (if on AWS EC2)</td>
-    </tr>
-    <tr>
-      <td>
-        <code>{'${attr.platform.aws.instance-life-cycle}'}</code>
-      </td>
-      <td>Instance lifecycle (e.g. spot, on-demand) of the client (if on AWS EC2)</td>
-    </tr>
-    <tr>
-      <td>
-        <code>{'${attr.platform.aws.instance-type}'}</code>
-      </td>
-      <td>Instance type of the client (if on AWS EC2)</td>
-    </tr>
-    <tr>
-      <td>
-        <code>{'${attr.platform.aws.placement.availability-zone}'}</code>
-      </td>
-      <td>Availability Zone of the client (if on AWS EC2)</td>
-    </tr>
-    <tr>
-      <td>
-        <code>{'${attr.os.name}'}</code>
-      </td>
-      <td>
-        Operating system of the client (e.g. <code>ubuntu</code>,{' '}
-        <code>windows</code>, <code>darwin</code>)
-      </td>
-    </tr>
-    <tr>
-      <td>
-        <code>{'${attr.os.version}'}</code>
-      </td>
-      <td>Version of the client OS</td>
-    </tr>
-  </tbody>
-</table>
+| Property                                               | Description                                                                                                                                            |
+| ------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `{'${attr.cpu.arch}'}`                                 | CPU architecture of the client (e.g. `amd64`, `386`)                                                                                              |
```

Comment:
In the preview, I'm seeing these as brackets render:

### [Upstream (HTML syntax)](https://www.nomadproject.io/docs/runtime/interpolation)

<img width="976" alt="CleanShot 2022-10-18 at 10 05 56@2x" src="https://user-images.githubusercontent.com/4624598/196453246-45be8057-cd58-4480-bb71-ab6850ca33ed.png">

### [Preview from this PR](https://nomad-hvhp9clzl-hashicorp.vercel.app/docs/runtime/interpolation)

<img width="942" alt="CleanShot 2022-10-18 at 10 05 46@2x" src="https://user-images.githubusercontent.com/4624598/196453349-65178927-19d6-47d9-ac64-cfcf730e3640.png">

Can we do something like this:

```suggestion
| `${attr.cpu.arch}`                                 | CPU architecture of the client (e.g. `amd64`, `386`)                                                                                              |
```

... or does that fail due to MDX or something?

Reply:
Good catch, I think it should be a-ok to remove these, one moment...